### PR TITLE
[FIX] google_calendar: increase debug when all events in a recurrence…

### PR DESCRIPTION
**Pull request by arj-odoo**
… are deleted.

Before this commit, a lot of events and mail followers would be deleted and recreated.
This PR commit aims to identify the issue and fix it.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by tfr-odoo** [comment]
This line https://github.com/odoo/odoo/blob/15.0/addons/google_calendar/models/calendar.py#L135 shouldn't also be replace with a _get_sync_partner for non recurrent event ?

File: addons/google_calendar/models/calendar.py [hunk 1]
```diff
@@ -132,7 +132,7 @@ def _odoo_attendee_commands(self, google_event):
         if google_event.exists(self.env):
             existing_attendees = self.browse(google_event.odoo_id(self.env)).attendee_ids
         attendees_by_emails = {tools.email_normalize(a.email): a for a in existing_attendees}
-        partners = self.env['mail.thread']._mail_find_partner_from_emails(emails, records=self, force_create=True)
+        partners = self._get_sync_partner(emails)
```
Comment (arj-odoo) [inline addons/google_calendar/models/calendar.py:135]: @tfr-odoo I replace the partner for single event here.

**Comment by tfr-odoo** [comment]
@robodoo r+

**Comment by tfr-odoo** [comment]
@robodoo rebase-ff

**Pull request closed without merging**